### ISSUE TITLE
Package Request: spm

### ISSUE THREAD

**Issue by silvernode** [voidlinux/void-packages#12243]
## Description  

SPM  "A simple commandline package manager that installs AppImages and precompiled tar archives."

## Other Comments

This software is a series of shell scripts that does a superb job at managing Appimages and tar archives to the point that I think it could be a great addition to Void. It connects to "AppimageHub" giving access to many applications that are not currently in the Void repository or never will be. As an example I use Wire messenger as my primary messaging platform. It's written with Electron and spm makes it simple to install and get running very quickly. Hopefully someone sees it's value like I do and considers packaging it as I think it will become a great addition to void-packages.  

**License:** GPLv2  
**Github Project:** https://github.com/simoniz0r/spm  
**Official Website:** http://www.simonizor.gq/ (domain recently expired)
**AppimageHub:** https://appimage.github.io/apps/ 

<a href="https://asciinema.org/a/OY8FwYH8I9j590or2iPTiid0O" target="_blank"><img src="https://asciinema.org/a/OY8FwYH8I9j590or2iPTiid0O.png" /></a>

**Comment by silvernode** [comment]
I meant to tag this as a request but can't seem to go back and edit tags. Sorry about that.

**Comment by maxice8** [comment]
this package doesn't inspire confidence.

#!/bin/bash as shebang
no religious use of quoting
no buildsystem ( not even a makefile )
build-spm downloads debian tarball apparently
the release tarball actually extracts to a tarball, what the fuck

**Comment by maxice8** [comment]
yeah sorry no, the scripts are poorly written, they require function scripts wrappers that should be in /usr/libexec/spm or /usr/share/spm to be in /usr/bin

**Comment by silvernode** [comment]
Wow I don't how I didn't notice the missing shebang in there. You're right, while it may function well on the end user side of things, the more I look into the scripts the more scratch my head.
